gutter/django/forms.py

```diff
@@ -196,7 +196,11 @@ def is_valid(self):
 
     def save(self, gutter_manager):
         switch = self.switch.to_object
-        switch.conditions = self.conditions.to_objects
+        try:
+            switch.conditions = self.conditions.to_objects
+        except AttributeError:
+            switch.conditions = self.conditions
```

Comment:
I wonder if it makes sense to use a different attribute here on `SwitchFormManager` instead of `conditions`, which has an existing meaning.